frontend/src/pages/Question.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+function Question() {
+  const [question, setQuestion] = useState([]);
+  const { categoryId } = useParams();
+  useEffect(() => {
+    axios
+      .get(
+        `${
+          import.meta.env.VITE_BACKEND_URL ?? "http://localhost:5000"
+        }/categories/${categoryId}/questions`
+      )
+      .then((response) => {
+        setQuestion(response.data);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }, []);
+
+  return (
+    <div>
+      {question.map((categorie) => (
```

Comment:
You are displaying multiples questions in a component named `Question`, it's not consistent. Rename your file and your component or display only one question